src/main/java/com/omerinfo/MySqlConnector.java

```diff
@@ -16,7 +16,7 @@ public class MySqlConnector {
     private static final String USER = "root";
 
     // The password for the database connection.
-    private static final String PASSWORD = "Pavel443";
```

Comment:
My local password is 'root' You can keep your password 'Pavel443' on your local.